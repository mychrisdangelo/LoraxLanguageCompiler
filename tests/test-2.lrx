
int a;//0

bool chris(int d){
	return false;
}

float zhaarn(float m){
	return 4.3;
}

char kira(float f){
	return 'c';
}

int main()
{
	return 0;
}


