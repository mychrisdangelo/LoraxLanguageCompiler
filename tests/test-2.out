#include "lrxlib.h"
int a_0;

int main();
char kira(float);
float zhaarn(float);
bool chris(int);

int main()
{
int __tmp_int_7 = 0;
int __tmp_int_6 = 0;

__tmp_int_7 = 0;

return __tmp_int_7;
return __tmp_int_6;

}
char kira(float f_3)
{
char __tmp_char_5 = '\0';
char __tmp_char_4 = '\0';

__tmp_char_5 = 'c';

return __tmp_char_5;
return __tmp_char_4;

}
float zhaarn(float m_2)
{
float __tmp_float_3 = 0.0;
float __tmp_float_2 = 0.0;

__tmp_float_3 = 4.3;

return __tmp_float_3;
return __tmp_float_2;

}
bool chris(int d_1)
{
bool __tmp_bool_1 = false;
bool __tmp_bool_0 = false;

__tmp_bool_1 = false;

return __tmp_bool_1;
return __tmp_bool_0;

}
